xbmc/linux/LinuxTimezone.h

```diff
@@ -59,6 +60,7 @@ class CLinuxTimezone : public ISettingCallback, public ISettingsHandler
    std::map<std::string, std::string> m_countriesByTimezoneName;
 };
 
-extern CLinuxTimezone g_timezone;
+XBMC_GLOBAL_REF(CLinuxTimezone,g_timezone);
+#define g_timezone XBMC_GLOBAL_USE(CLinuxTimezone)
```

Comment:
Ok. Then how about a method like `static CLinuxTimezone& CLinuxTimezone::getInstance()` which creates an instance of `CLinuxTimezone` on the first call?